VideoWeb/VideoWeb/ClientApp/src/app/services/conference/video-control-cache.service.ts

```diff
@@ -18,6 +18,11 @@ export class VideoControlCacheService {
         private logger: LoggerService
     ) {
         this.conferenceService.currentConference$.subscribe(conference => {
+            if (!conference) {
+                this.logger.warn(`${this.loggerPrefix} No conference loaded. Skipping loading of hearing state for conference`);
+                return;
+            }
```

Comment:
Could you add a test when conference is falsy that checks that it doesn't go past this point?